bee-message/src/byte_cost.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::output::Output;
+
+use packable::PackableExt;
+
+/// Specifies the current parameters for the byte cost computation.
+#[derive(Clone)]
+pub struct ByteCostConfig {
+    /// Cost in tokens coins per virtual byte.
+    pub byte_cost: u64,
+    /// The weight factor used for key fields in the ouputs.
+    pub weight_for_key: u64,
+    /// The weight factor used for data fields in the ouputs.
+    pub weight_for_data: u64,
+}
+
+impl Default for ByteCostConfig {
+    fn default() -> Self {
+        Self {
+            byte_cost: 1,
```

Comment:
Shouldn't this be set to `500`? See the default value here https://github.com/Wollac/protocol-rfcs/blob/protocol-parameters/tips/TIP-0022/tip-0022.md#detailed-design